policy/release/olm.rego

```diff
@@ -215,6 +215,42 @@ deny contains result if {
 	result := lib.result_helper_with_term(rego.metadata.chain(), [image.str(unmatched_image.ref)], image.str(unmatched_image.ref))
 }
 
+# METADATA
+# title: Images referenced by OLM bundle are from allowed registries
+# description: >-
+#   Each image referenced by the OLM bundle should match an entry in the list of prefixes
+#   defined by the rule data key `allowed_registry_prefixes` in your policy configuration.
+# custom:
+#   short_name: allowed_registries
+#   failure_msg: The %q CSV image reference is not from an allowed registry.
+#   solution: >-
+#     Use image from an allowed registry, or modify your
+#     xref:ec-cli:ROOT:configuration.adoc#_data_sources[policy configuration] to include additional registry prefixes.
+#   collections:
+#   - redhat
+#   effective_on: 2024-0901T00:00:00Z
```

Comment:
Typo
```suggestion
#   effective_on: 2024-09-01T00:00:00Z
```

Reply:
Nice catch, thanks

Reply:
Fixed. Also fixed in `antora/docs/modules/ROOT/pages/release_policy.adoc`